README.md

```diff
@@ -58,13 +58,18 @@ void loop() {
     bleKeyboard.write(KEY_MEDIA_PLAY_PAUSE);
 
     delay(1000);
-
-    Serial.println("Sending Ctrl+Alt+Delete...");
+    
+   //
+   // Below is an example of pressing multiple keyboard modifiers 
+   // which by default is commented out - for obvious reasons. 
```

Comment:
Can you take out the "for obvious reasons" please. It is not obvious if you ask me.

Reply:
Removed. Weird about the code change, this is patch-2,  patch-1  was the code change.